.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -1,8 +1,9 @@
 name: Bug report
```

Comment:
Interesting; if i click "view file", github shows a big fat warning, but perhaps it's just a glitch / github being off because of the rename or something?

<img width="533" alt="Screenshot 2025-04-17 at 22 21 12" src="https://github.com/user-attachments/assets/d4b1dd03-f464-486b-9bff-b979bddd8ed7" />